src/blueprint/into_schema.rs

```diff
@@ -133,6 +133,42 @@ fn create(blueprint: &Blueprint) -> SchemaBuilder {
   schema
 }
 
+pub(crate) fn create_dumb_schema(blueprint: &Blueprint) -> SchemaBuilder {
```

Comment:
It required to alter the behavior of the `create` function, which might lead to bugs, and updating the blueprint would still not prevent the real execution of the requests, which is why I added this method. But it reuses most of the previous code. It only makes _null_ the resolver.

Reply:
I totally get why you've added the method. 

But to express my concerns in more details: there is no code reuse here because you've actually created new code instead of using previous one. We can only hope that code acts the same way the original code works. For now it could be easy to see it's ok, but such code tends to diverge with time and it needs to be supported. Moreover, see the codecov comments in your pr? It shows this function is not covered by tests, so we probably need to create additional set of tests to make sure `create_dumb_schema` works as expected that should be same as tests for normal workflow